desktop/src/main/java/bisq/desktop/main/dao/governance/result/VoteResultView.fxml

```diff
@@ -29,4 +31,7 @@
     <columnConstraints>
         <ColumnConstraints percentWidth="100"/>
     </columnConstraints>
+    <HBox GridPane.rowIndex="8" GridPane.columnSpan="3" alignment="BOTTOM_RIGHT">
```

Comment:
We do not use FXML anymore beside of the containers (one day we want to get rid completely of it) as it has bad performance and is too limited for more dynamic content. There are still relicts but new code we want to have in Java only. See FormBuilder as out Util for UI components.